spec/tic_tac_toe/board_spec.clj

```diff
@@ -0,0 +1,7 @@
+(ns tic-tac-toe.board-spec
+  (:require [speclj.core :refer :all]
+            [tic-tac-toe.board :refer :all]))
+
+(describe "a board"
+  (it "returns an empty board"
+    (should= '(- - - - - - - -) (empty-board) )))
```

Comment:
Think about namespacing here, so it's obvious where the function is coming from. board/empty-board for example.